osu.Game/Screens/OnlinePlay/OnlinePlaySongSelect.cs

```diff
@@ -115,45 +114,47 @@ protected override void LoadComplete()
                 Freestyle.Value = initialItem.Freestyle;
             }
 
-            Mods.BindValueChanged(onModsChanged);
+            Mods.BindValueChanged(onGlobalModsChanged);
             Ruleset.BindValueChanged(onRulesetChanged);
             Freestyle.BindValueChanged(onFreestyleChanged, true);
 
+            if (initialItem == null)
+            {
+                // Enable all free mods if we're creating a new playlist item.
+                // Todo: This needs to be scheduled because mods aren't available until the nested LoadComplete(). Can we do this any better?
```

Comment:
Is this todo supposed to be here still?

Reply:
I haven't looked into this so it's a "yes" for the time being. Unless you have any suggestions...